improver/wxcode/wxcode_decision_tree.py

```diff
@@ -247,8 +245,8 @@ def wxcode_decision_tree():
                 [['probability_of_lwe_snowfall_rate_above_threshold',
                   'probability_of_rainfall_rate_above_threshold']],
             'diagnostic_gamma': [1.],
-            'diagnostic_thresholds': [[AuxCoord(0.1, units='mm hr-1'),
-                                       AuxCoord(0.1, units='mm hr-1')]],
+            'diagnostic_thresholds': [[(0.1, 'mm hr-1'),
+                                       (0.1, 'mm hr-1')]],
             'diagnostic_conditions': ['above', 'above']},
```

Comment:
should this be `[['above', 'above']]` instead?

Reply:
Yes, I think so (for all of these). I think the behaviour of `expand_nested_lists` is permitting these inconsistencies.

Reply:
There appear to be quite a few inconsistencies in how we have bracketed things. We can either fix them here, or review the lot all at once. Happy for this PR to go in unchanged and I'll open a little fix ticket to tidy up the definitions at a later date.

Reply:
I agree.